cmd/monaco/main.go

```diff
@@ -336,3 +344,40 @@ func getDownloadCommand(fileReader util.FileReader) cli.Command {
 	}
 	return command
 }
+func getInitialiseCommand() cli.Command {
+	command := cli.Command{
+		Name:      "init",
+		Usage:     "initialise demo project and folders. Optionally pass a list of APIs to create demo folders or leave blank for all",
+		UsageText: "init [command options]",
+		Before: func(c *cli.Context) error {
+			err := util.SetupLogging(c.Bool("verbose"))
+
+			if err != nil {
+				return err
+			}
+
+			util.Log.Info("Dynatrace Monitoring as Code v" + version.MonitoringAsCode)
+
+			return nil
+		},
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "createSpecificAPI",
+				Usage:   "Comma separated list of API's to create",
+				Aliases: []string{"c"},
+			},
+		},
+		Action: func(ctx *cli.Context) error {
+			var workingDir string
+
+			if ctx.Args().Present() {
```

Comment:
I guess we should handle the case somebody provides more than one argument. 
If you ask me, we should then return an error telling the user only to provide one.

Reply:
Happy to implement if only I knew how :)

Reply:
Have a look at how it is done on line 263 :)

Reply:
Sorry I meant that you test if there are more than one argument present.

Having one argument to override the working dir is actually quite good.